src/SprykerEco/Zed/Payone/Business/PayoneBusinessFactory.php

```diff
@@ -413,4 +761,140 @@ public function createOrderPriceDistributor(): OrderPriceDistributorInterface
     {
         return new OrderPriceDistributor();
     }
+
+    /**
+     * @return \SprykerEco\Zed\Payone\Business\Payment\DataMapper\StandartParameterMapperInterface
+     */
+    public function createStandartParameterMapper(): StandartParameterMapperInterface
+    {
+        return new StandartParameterMapper($this->createKeyHashGenerator(), $this->createModeDetector());
+    }
+
+    /**
+     * @return \SprykerEco\Zed\Payone\Business\Payment\DataMapper\PayoneRequestProductDataMapperInterface
+     */
+    public function createPayoneRequestProductDataMapper(): PayoneRequestProductDataMapperInterface
+    {
+        return new PayoneRequestProductDataMapper(
+            $this->createProductsMapper(),
+            $this->createShipmentMapper(),
+            $this->createDiscountMapper(),
+        );
+    }
+
+    /**
+     * @return \SprykerEco\Zed\Payone\Business\Payment\DataMapper\ProductMapperInterface
+     */
+    public function createProductsMapper(): ProductMapperInterface
+    {
+        return new ProductMapper();
+    }
+
+    /**
+     * @return \SprykerEco\Zed\Payone\Business\Payment\DataMapper\ShipmentMapperInterface
+     */
+    public function createShipmentMapper(): ShipmentMapperInterface
+    {
+        return new ShipmentMapper();
+    }
+
+    /**
+     * @return \SprykerEco\Zed\Payone\Business\Payment\PaymentMapperReader
+     */
+    public function createPaymentMapperReader(): PaymentMapperReader
+    {
+        $paymentMapperReader = new PaymentMapperReader(
+            $this->createSequenceNumberProvider(),
+            $this->createUrlHmacGenerator()
+        );
+
+        foreach ($this->getAvailablePaymentMethods() as $paymentMethod) {
+            $paymentMapperReader->registerPaymentMethodMapper($paymentMethod, $this->getStandardParameter());
+        }
+
+        return $paymentMapperReader;
+    }
+
+    /**
+     * @return \SprykerEco\Zed\Payone\Business\Payment\DataMapper\DiscountMapperInterface
+     */
+    public function createDiscountMapper(): DiscountMapperInterface
+    {
+        return new DiscountMapper();
+    }
+
+    /**
+     * @return \SprykerEco\Zed\Payone\Business\Payment\DataMapper\OrderHandlingMapperInterface
+     */
+    public function createOrderHandlingMapper(): OrderHandlingMapperInterface
+    {
+        return new OrderHandlingMapper();
+    }
+
+    /**
+     * @return \Symfony\Component\HttpFoundation\RequestStack
+     */
+    public function getRequestStack()
```

Comment:
* Add a type hint.
* All dependency getter methods should go last in the class.

Reply:
++